### DISCUSSION TITLE
How do I convert a multipage PDF to single GIF files? I don't want to create an animated GIF.

### DISCUSSION THREAD

**Discussion by futuremotiondev** [ImageMagick/ImageMagick#5558] [Help]
I am using this command:

`magick -colorspace rgb -density 200 input.pdf -resize 100% output.gif`

But this by default creates an animated GIF. I don't want this. I want to create separate GIF images for each page. How do I override the default behavior of combining output to an animated gif?

Any help greatly appreciated!

**Comment by fmw42** [comment]
`magick -colorspace rgb -density 200 input.pdf -resize 100% +adjoin output.gif`

or

`magick -colorspace rgb -density 200 input.pdf -resize 100% +adjoin output_%02d.gif`

%02d will auto-number with leading 0. (i.e., 00, 01 -- 09, 10 -- 99)
